### DISCUSSION TITLE
Export documentation HTML?

### DISCUSSION THREAD

**Discussion by JASKevinWhite** [danielgtaylor/huma#425] [Q&A] [answered]
I feel like I'm missing something simple here, and if so, my apologies.

I've used Huma to create an API, and it has created a wonderful /docs page, which I love. But I'd like to export the page so that it can be delivered to a customer or published on a web server. I can't find anything in the Huma docs on how to do so.

Is this possible?

Thank you!

**Comment by danielgtaylor** [comment, marked as answer]
@JASKevinWhite the docs are actually a very small static HTML page that dynamically loads the OpenAPI from your service. Check here:

https://github.com/danielgtaylor/huma/blob/main/api.go#L448-L470

You can copy that, and if you want can also download the OpenAPI from your service, point the static docs to the static OpenAPI, and you are good to go. You probably just want to update it on each release via some automated process if you can.
